src/microbe_stage/Microbe.cs

```diff
@@ -1061,6 +1178,44 @@ private void RefreshProcesses()
         processesDirty = false;
     }
 
+    private void RefreshEnzymes()
+    {
+        if (enzymes == null)
+        {
+            enzymes = new Dictionary<Enzyme, int>();
+        }
+        else
+        {
+            enzymes.Clear();
+        }
+
+        if (organelles == null)
+            return;
+
+        // Cells have a minimum of at least one unit of lipase enzyme
+        enzymes[lipase] = 1;
+
+        foreach (var organelle in organelles.Organelles)
+        {
+            if (organelle.StoredEnzymes == null)
+                continue;
+
+            foreach (var enzyme in organelle.StoredEnzymes)
+            {
+                // Should have a minimum of unit of enzyme
+                if (enzyme.Value <= 0)
+                    continue;
+
+                if (!enzymes.TryGetValue(enzyme.Key, out int count))
+                    count = 0;
```

Comment:
The out variable is actually default initialized to 0 if not found, so this if and explicit 0 set should be unnecessary.